server/index.js

```diff
@@ -152,7 +160,11 @@ will increase speed of future builds, so please be patient.'
     watch: [`${bpanelConfig.prefix}/config.js`],
     env: {
       BPANEL_PREFIX: bpanelConfig.prefix,
-      BPANEL_SOCKET_PORT: bsockPort
+      BPANEL_SOCKET_PORT: bsockPort,
```

Comment:
What is the difference between defining these here and passing them into the webpack config itself?